package.json

```diff
@@ -51,7 +51,7 @@
     "rollup": "^0.41.6",
     "standard-version": "^4.0.0",
     "tslint": "^5.1.0",
-    "typescript": "^2.3.2",
+    "typescript": "^2.4.0",
```

Comment:
are, you're right it installs 2.4.1 now, it was installing old version in one of the projects  because of the package-lock.json file..